### PR TITLE
Update Spanish translation

### PR DESCRIPTION
**Pull request by DanielSevillano**

### CODE REVIEW

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -68,7 +68,7 @@
     <string name="theme">Tema</string>
     <string name="theme_mode">Modo de tema</string>
     <string name="shapes">FORMAS</string>
-    <string name="thumbnail_roundness">Miniaturas redondeadas</string>
+    <string name="thumbnail_roundness">Redondez de las miniaturas</string>
```
Comment (kikorb) [inline app/src/main/res/values-es/strings.xml:71]: I would say Miniaturas redondeadas sounds better.

**Comment by fast4x** [comment]
Merged

File: app/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -160,18 +160,18 @@
...
-    <string name="i18_medium">Medium</string>
-    <string name="i18_big">Big</string>
-    <string name="i18_none">None</string>
-    <string name="i18_light">Light</string>
-    <string name="i18_heavy">Heavy</string>
-    <string name="effects">Effects</string>
-    <string name="player_rotating_buttons">Rotating buttons</string>
-    <string name="player_enable_rotation_buttons">Enable rotation of buttons when clicked</string>
+    <string name="player_thumbnail_size">Tamaño de las miniaturas</string>
+    <string name="restarting_rimusic_is_required">Se requiere un reinicio</string>
```
Comment (kikorb) [inline app/src/main/res/values-es/strings.xml:164]:
```suggestion
    <string name="restarting_rimusic_is_required">Se requiere reiniciar</string>
```

**Comment by kikorb** [comment]
I came late to the PR

File: app/src/main/res/values-es/strings.xml (hunk 1 above)
Comment (fast4x) [reply app/src/main/res/values-es/strings.xml:71]: Quiet, there are other translations coming.
Statistics and other option settings.
:)

Comment (DanielSevillano) [reply app/src/main/res/values-es/strings.xml:71]: > I would say Miniaturas redondeadas sounds better.

I agree, but I don't think it describes well what it does. We could change it to "Curvatura de las miniaturas", it sounds better.

Comment (fast4x) [reply app/src/main/res/values-es/strings.xml:71]: Sure

**Pull request merged**